UpdateDependencies.ps1

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Any reason to not just call it run.ps1?

Reply:
I'm a little concerned someone would expect it to work like `run.cmd`, like using the `config.json` settings. Making a more general name sounds reasonable, but ideally I'd like something that somehow gets across that it's "only" msbuild, and that it runs on .NET Core rather than full framework.

There's also a kind of symmetry that exists if we use this name, since both release/1.0.0 and release/1.1.0 will have both UpdatePublishedVersions.ps1 and UpdateDependencies.ps1, where release/1.0.0's are plain old powershell scripts and release/1.1.0's both use MSBuild.

Reply:
I'm find with with not making it attractive to use but the name is definitely miss-leading. Should we at least have it have the correct project to build or do you plan to use it as a wrapper for other dotnet cli msbuild invocations.

Reply:
Hmm, good point--since we have to make subscriptions.json changes anyway, I should move the project and target here. I don't plan on using this script for anything general.